style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  background-color: #191765;
+}
+
+.head {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
- [ ] Kindly add the background image (ellipse at the top left) as per the Figma design.
- Your design:
<img width="308" alt="Screenshot 2022-03-17 at 11 44 16 AM" src="https://user-images.githubusercontent.com/70269507/158772944-61f2cdad-7347-432a-b62b-5c1419be2b69.png">

- Figma design:
<img width="267" alt="Screenshot 2022-03-17 at 11 43 41 AM" src="https://user-images.githubusercontent.com/70269507/158772982-17c12eee-0652-4c17-906a-cc433a128619.png">